framework/auth/core.py

```diff
@@ -2,40 +2,16 @@
 
 import datetime as dt
 
-import framework
 import itertools
 import logging
-import re
-import urllib
-import urlparse
-from copy import deepcopy
-from framework import analytics
 
-import bson
-import itsdangerous
-import pytz
 from django.utils import timezone
-from flask import Request as FlaskRequest
-from framework.addons import AddonModelMixin
-from framework.auth import signals, utils
-from framework.auth.exceptions import (ChangePasswordError, ExpiredTokenError,
-                                       InvalidTokenError,
-                                       MergeConfirmedRequiredError,
-                                       MergeConflictError)
-from framework.bcrypt import check_password_hash, generate_password_hash
-from framework.exceptions import PermissionsError
-from framework.guid.model import GuidStoredObject
-from framework.mongo import get_cache_key
 from framework.mongo.validators import string_required
-from framework.sentry import log_exception
 from framework.sessions import session
-from framework.sessions.model import Session
-from framework.sessions.utils import remove_sessions_for_user
-from modularodm import Q, fields
-from modularodm.exceptions import (NoResultsFound, QueryException,
-                                   ValidationError, ValidationValueError)
+from modularodm import Q
```

Comment:
Presumably the same one to [remove MODMCompatibilityQuerySet](https://openscience.atlassian.net/browse/OSF-7863)